./error_fun_ret4.nit:17,1--21,10: Control error: Reached end of function (a 'return' with a value was expected).
